snuba/clickhouse/translators/snuba/mappers.py

```diff
@@ -141,11 +142,23 @@ def attempt_map(
             expression.column.table_name == self.from_column_table
             and expression.column.column_name == self.from_column_name
         ):
-            return build_mapping_expr(
+            key = expression.key.accept(children_translator)
+            resolved_expr = build_mapping_expr(
```

Comment:
Why even build this is `self.nullable == True`?

Reply:
removed